programs/ssl/ssl_mail_client.c

```diff
@@ -20,6 +20,7 @@
 /* Enable definition of gethostname() even when compiling with -std=c99. Must
  * be set before config.h, which pulls in glibc's features.h indirectly.
  * Harmless on other platforms. */
+
```

Comment:
Is this an artefact of the script?

Reply:
unlikely, more like my mistake during manual modifcations.

Reply:
I am not sure what happened here. In PR summary this is an added empty line, but when I go to https://github.com/ARMmbed/mbedtls/pull/4511/commits/55768f26fc999b560efc2b13b0d33faa43b24bbf#diff-3624f543459744d6090d2525ebc8fc9a2df71caca086265edc00dd3649cadf9cR23
then the added line is not there :/